feat: complete redesign of tinycl

shlokj opened the pull request:
Redid the tinycl site to look better - replicated the design made on the figma.

Test it out at https://deploy-preview-32--tinycl.netlify.app/

I had to add .prettierrc since the existing linting configuration conflicts with prettier's default formatting rules.

shlokj committed:
complete redesign of website along with footer

shlokj commented on styles/Page.module.scss:
```diff
@@ -4,41 +4,54 @@
+  font-size: 32px;
 }
 
-@media screen and (max-width: 600px) {
```
i removed this conditional styling since the styles that are changed are no longer used

darren2hang reviewed (changes requested):
Great work! The preview looks amazing.
I haven't had a chance to take a detailed dive into your code, but I have a few suggestions.
Make sure to run the linter `yarn lint` and fix any linter errors.
When the browser is between 601px and 757px width, the links at the footer get a bit cut off. In the screenshot below, the EMAIL link isn't fully displayed
<img width="714" alt="Screenshot 2023-04-01 at 1 13 04 AM" src="https://user-images.githubusercontent.com/56695604/229274365-43c1cab4-42ed-4287-9cf9-d364230a21f0.png">

I think some potential fixes are changing the media query max-width attribute from 600px to 760px or making the font size scale with the page width

shlokj commented:
Great catch, thanks! I changed the media query max-width attribute to 800px (wanted to give more room than 760, which was near 757, the exact limit) and now it seems to work.

`yarn lint` returned minor problems only with one of the files I changed; I've fixed them now.

darren2hang commented on styles/Page.module.scss:
```diff
@@ -4,41 +4,54 @@
-  }
+#text-about {
+  font-family: 'Lato', sans-serif;
+  margin-top: -12px; // TODO: find a better way to do this
```
```suggestion
  margin-top: 20px;
```

shlokj replied:
This doesn't really solve the issue - it just increases the space between the two

darren2hang commented on styles/Page.module.scss:
```diff
@@ -4,41 +4,54 @@
-  }
+#atcl {
+  font-family: 'Mulish', sans-serif;
+  font-size: 32px;
```
```suggestion
  font-size: 32px;
  margin-bottom: 0px;
```

darren2hang reviewed (dismissed):
A made a suggestions on how to address ur TODO of getting rid of the negative margin.
Otherwise, the code looks good to me. I like the TCLButton and the window.open() function!
I'll leave it up to you if you want to change the CSS. Feel free to squash and merge whenever you want

darren2hang reviewed (approved):
Thanks for the awesome work! LGTM!
